### PR TITLE
#7233 Update versions for 2021.02.xx

### PR DESCRIPTION
**Pull request by offtherailz**
## Description
This PR updates the pom.xml files with the stable versions of libs and fixes the versions of `mapstore-services`, `mapstore-web` etc...

### CODE REVIEW

**Comment by offtherailz** [comment]
Maybe we should split this in two:
- Fix versions of dependencies
- Fix versions of modules (and deploy on maven.geo-solutions.it) the versions.
What do you think @mbarto

File: project/custom/templates/web/pom.xml [hunk 1]
```diff
@@ -20,23 +20,23 @@
     <!-- MapStore backend -->
     <dependency>
       <groupId>it.geosolutions.mapstore</groupId>
-      <artifactId>mapstore-backend</artifactId>
-      <version>1.0-SNAPSHOT</version>
+      <artifactId>mapstore-service</artifactId>
```
Comment (mbarto) [inline project/custom/templates/web/pom.xml:23]:
```suggestion
      <artifactId>mapstore-services</artifactId>
```

**Review by mbarto — CHANGES_REQUESTED** [review]
I found a typo. Except for that, it looks good.

**Review by mbarto — APPROVED** [review]

**Comment by offtherailz** [comment]
Thank you. Copying by hand, maybe passed to me.

**Pull request merged**